### ISSUE TITLE
Browser Cache Full Prevents App Load

### ISSUE THREAD

**Issue by andymwta** [whatcomtrans/WTA.Scheduler#70]
If user's browser cache is full, app will just spin.
Opening app in another browser is fine.
Clearing browser cache corrects problem.

detect if user cache is full and give a hint

**Comment by andymwta** [comment]
A few things changed:
The wta.appcache file being served from S3 did not have correct headers. They are now:
Content-Type text/cache-manifest (tells browser it's a manifest file, most browsers will automatically understand this but IE7-9 would not)
Cache-Control no-cache (prevents the manifest file itself from being cached)

The cache from the app was hovering around 13MB, which can exceed the shared cache limit imposed by web-kit browsers. I've removed the non-gz version of data_trips.js from the manifest file, which has reduced the cache size to 7.6MB.
This means that if a user's browser does not support gzipped files (which is extremely unlikely if their browser supports manifest in the first place), the network section of the manifest specifies "*", which will force the data_trips.js file to download. This will impact their performance but the cases should be very limited.

I've also added code at the beginning of schedules.html that detects if a new version of the manifest is available and forces a fresh load. This gives us the ability to immediately update user's browsers if the need arises.

In addition, I've added documentation to oneNote about making updates to the scheduling app with regards to required headers.

I have no means of testing if this fixed the problem now; will be asking Becky to try and load the app on the device that produced this issue this evening and see what results she gets.
